docs/uma-tokenholders/approved-collateral-currencies.md

```diff
@@ -3,81 +3,92 @@ title: Approved collateral currencies
-|VSP| 50 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|FRAX| 400 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|DEXTF| 2250 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|ORN| 75 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|BOND| 10 | [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|PUNK-BASIC| 0.015| [UMIP-82](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-82.md)
-|LUSD| 400 | [UMIP-86](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-86.md)
-|iFARM| 7.5 | [UMIP-88](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-88.md)
-|yvUSDC| 400 | [UMIP-93](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-93.md)
-|UST| 400 | [UMIP-94](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-94.md)
-|BNT| 135 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|vBNT| 400 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|BAND| 75 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|SDT| 450 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|KP3R| 5 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|CREAM| 4 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|CHAIN| 5000 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|ERN| 70 | [UMIP-96](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-96.md)
-|OPEN| 1000 | [UMIP-102](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-102.md)
-|DFX| 900 | [UMIP-104](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-104.md)
-|BASK| 1.75 | [UMIP-105](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-105.md)
-|GYSR| 2000 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|MPH| 15 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|APW| 400 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|SNOW| 60 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|NDX| 100 | [UMIP-111](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-111.md)
-|Total| No of Approved collateral currencies | [73](https://docs.umaproject.org/uma-tokenholders/approved-collateral-currencies)
+|Collateral Currency| Final Fee | UMIP | Network|
+|:-------| :-----------| :-----------| :-----------|
+|WETH| 0.2 |  [UMIP-10](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-10.md), [UMIP-113](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-113.md) | [Ethereum](https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2), [Polygon](https://polygonscan.com/address/0x7ceb23fd6bc0add59e62ac25578270cff1b9f619)
```

Comment:
It looks like you are missing end bars for all of the entries in this table, which makes the hyperlinks not work correctly. I think you just need to add "|" to the end of each entry.